elements/pfe-spinner/test/pfe-spinner.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { expect, html } from '@open-wc/testing';
+import { createFixture } from '@patternfly/pfe-tools/test/create-fixture.js';
+import { PfeSpinner } from '@patternfly/pfe-spinner';
+
+describe('<pfe-spinner>', function() {
+  it('should upgrade', async function() {
+    const element = await createFixture<PfeSpinner>(html`<pfe-spinner>Loading...</pfe-spinner>`);
+    expect(element, 'pfe-spinner should be an instance of PfeSpinner')
+      .to.be.an.instanceOf(customElements.get('pfe-spinner'))
+      .and
+      .to.be.an.instanceOf(PfeSpinner);
+  });
+
+  it('should properly initialize the component', async () => {
```

Comment:
```suggestion
  it('should properly initialize the component', async function() {
```

[mocha tests should not use arrow functions](https://mochajs.org/#arrow-functions)